gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTableFilter.java

```diff
@@ -82,4 +84,23 @@ public KnoxShellTable greaterThan(Comparable<? extends Object> comparable) throw
     return filter(s -> s.compareTo(comparable) > 0);
   }
 
+  @SuppressWarnings("unchecked")
+  public KnoxShellTable lessThan(Comparable<? extends Object> comparable) throws KnoxShellTableFilterException {
+    return filter(s -> s.compareTo(comparable) < 0);
+  }
+
+  @SuppressWarnings("unchecked")
+  public KnoxShellTable greaterThanOrEqualTo(Comparable<? extends Object> comparable) throws KnoxShellTableFilterException {
+    return filter(s -> s.compareTo(comparable) > 0 || s.compareTo(comparable) == 0);
+  }
+
+  @SuppressWarnings("unchecked")
+  public KnoxShellTable lessThanOrEqualTo(Comparable<? extends Object> comparable) throws KnoxShellTableFilterException {
+    return filter(s -> s.compareTo(comparable) < 0 || s.compareTo(comparable) == 0);
```

Comment:
Should be `.equals()` as well for gt and lt

Reply:
Updated in latest patch.